tests/functional/processes/file/test_fulfillment_process.py

```diff
@@ -0,0 +1,73 @@
+from processes import ClusterProcess, FulfillURLManifestProcess
+from model import Record, Item
+import json
+import os
+from sqlalchemy.orm import joinedload
+import pytest
+
+def test_fulfillment_process(db_manager, s3_manager, seed_test_data):
+
+    limited_access_record_uuid = seed_test_data['manifest_record_uuid']
```

Comment:
We can then mass in the limited_access_record_uuid in as a param and don't have to declare it on line 10